src/ChannelServer/World/Character.cs

```diff
@@ -528,7 +612,7 @@ public int MaxMAtk
 		{
 			get
 			{
-				var baseValue = 20;
+				var baseValue = 22;
```

Comment:
Where did you get these 22s from? The client files (calc_property_pc) uses 20 o.o

Reply:
Looks like its out-dated, the http://www.tosbase.com/tools/stat-calculator/ use the same as the calc_property_pc and its always 2 lower then on the offi server.

Reply:
Nevermind, the problem was I didn't thought about the Collection bonus, using a new account now.